docs/general/emulator-support-and-issues.md

```diff
@@ -347,7 +349,6 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - Limited microphone support
 - ✅ libretro core: **DeSmuME**
 - ✅ libretro core: **DeSmuME 2015**
-- ✅ libretro core: **melonDS**
```

Comment:
[We support the melonDS libretro core](https://docs.libretro.com/guides/retroachievements/#nintendo-ds). Please revert.